fares-data-build-tool/create-data
# feat: Updating FareTable Name

## Description

Anywhere a FareTable is created it should have a name with this structure:

FareTable for {productName}

## Testing instructions

Check the generated netex to see is FareTable.Name is in this format FareTable for {productName}
